v2/component_analyses.yaml

```diff
@@ -0,0 +1,288 @@
+openapi: 3.0.1
+info:
+  title: Analytics Core API Specs for Version 2
+  version: "2"
+servers:
+- url: https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2
+  description: "Fabric8 Server"
+security:
+- tokenAuth: []
```

Comment:
what does this field referring to ? Please add description.

Reply:
@SriKrishnaPaparaju  This Field is for Authentication. If you can paste the raw code in Swagger Editor, then on Clicking Authorize, we will see something like this..

![image](https://user-images.githubusercontent.com/21026016/78219877-114c5b80-74de-11ea-8ab4-30b9db76f21e.png)

We can't (not allowed) to add a description here. 
So, `description` is added in tokenAuth definition.